src/commands/generate/redteam.ts

```diff
@@ -107,8 +107,13 @@ export async function doGenerateRedteam(options: RedteamGenerateOptions) {
     typeof s === 'string' ? { id: s } : s,
   );
 
-  logger.debug(`plugins: ${plugins.map((p) => p.id).join(', ')}`);
-  logger.debug(`strategies: ${strategyObjs.map((s) => s.id ?? s).join(', ')}`);
+  try {
+    logger.debug(`plugins: ${plugins.map((p) => p.id).join(', ')}`);
+    logger.debug(`strategies: ${strategyObjs.map((s) => s.id ?? s).join(', ')}`);
+  } catch (error) {
+    logger.error('Error logging plugins and strategies. One did not have a valid id.');
```

Comment:
How do we get here? 

nit, consider other ways to handle this such as throwing and exiting or listing a message like `x, y, z plugins are invalid`

Reply:
You get this if you have a blank line 🤣